src/main/java/bytetalker/exception/ByteTalkerException.java

```diff
@@ -0,0 +1,39 @@
+package bytetalker.exception;
+
+public class ByteTalkerException {
+    public static class TodoNoTaskException extends Exception {
+        public TodoNoTaskException(String errorMessage) {
+            super(errorMessage);
+        }
+    }
+
+    public static class TodoUnsupportedFormatException extends Exception {
+        public TodoUnsupportedFormatException(String errorMessage) {
+            super(errorMessage);
+        }
+    }
+
+    public static class DeadlineWrongFormatException extends Exception {
+        public DeadlineWrongFormatException(String errorMessage) {
+            super(errorMessage);
+        }
+    }
+
+    public static class DeadlineUnsupportedFormatException extends Exception {
```

Comment:
Seems like there is a `DeadlineWrongFormatException` and a `DeadlineUnsupportedFormatException`, could you share the reason to separate the two? It sounds like either can be used in the situation of a different format than expected!

Reply:
At the time of coding it, I was thinking of different errors that could rise, but realised that both errors are eventually the same! I will try to combine both exceptions